packages/cedar/src/Chart.ts

```diff
@@ -58,7 +58,14 @@ export interface ISeries {
 }
 
 export interface ILegend {
-  enable: boolean
+  enable?: boolean
+}
+
+export interface IStyles {
+  backgroundColor?: string,
+  backgroundAlpha?: number,
```

Comment:
I prefer `backgroundOpacity` if that's what this really is. re: `background: { color: 'red',  opacity: 0.5 }` that's fine, but also allow `background: 'red'`. Either way, you'll need a new interface like IColorOpacity and do what I'm asking this would be `background: string || IColorOpacity`.

What I don't like about that is that it's inconsistent w/ `textColor`. So unless there's a reason to make text overloaded (like we might later add font size or something), I'd say leave them as is (except changing `Alpha` to `Opacity`.

Reply:
Hmm, that is a good point. Realistically `textColor` should become `text: { color: 'red', fontWhatever: 'blahblah' }` that's if we are thinking forwards a bit. 

`backgroundColor` and `backgroundOpacity` could become `background: { color, opacity }` if we wanted it too. I don't think there would be any other additional things to add here. That being said: `Background color. You should set backgroundAlpha to >0 value in order background to be visible. We recommend setting background color directly on a chart's DIV instead of using this property.` < is part of the docs on the site. So maybe I should rework how this is implemented......